styles.css

```
@@ -55,23 +73,54 @@ ul {
   list-style-position: inside;
   width: 400px;
   padding-right: 50px;
+  padding-bottom: 12px;
 }
 
 ul li {
   padding-bottom: 8px;
 }
 
-.aboutMe {
+#aboutMe li {
+  padding-right: 90px;
+}
+
+#aboutMe {
```

See comment above and add a classname to your section and selector :) 
```suggestion
.aboutMe {
```